#ifndef __mm_basic_frame_control_s_control_rq_h__
#define __mm_basic_frame_control_s_control_rq_h__

#include "core/mm_core.h"

//////////////////////////////////////////////////////////////////////////
extern void hd_s_control_get_logger_rq(struct mm_net_tcp* net_tcp);
extern void hd_s_control_set_logger_rq(struct mm_net_tcp* net_tcp,mm_uint32_t level);
//////////////////////////////////////////////////////////////////////////

#endif//__mm_basic_frame_control_s_control_rq_h__